extensions/docbook-snippets/vsc-extension-quickstart.md

```diff
@@ -0,0 +1,23 @@
+# Welcome to your VS Code Extension
+
+## What's in the folder
+
+* This folder contains all of the files necessary for your extension.
+* `package.json` - this is the manifest file that defines the location of the snippet file and specifies the language of the snippets.
+* `snippets/snippets.code-snippets` - the file containing all snippets.
+
+## Get up and running straight away
+
+* Press `F5` to open a new window with your extension loaded.
+* Create a new file with a file name suffix matching your language.
+* Verify that your snippets are proposed on IntelliSense.
+
+## Make changes
+
+* You can relaunch the extension from the debug toolbar after making changes to the files listed above.
+* You can also reload (`Ctrl+R` or `Cmd+R` on Mac) the VS Code window with your extension to load your changes.
+
+## Install your extension
+
+* To start using your extension with Visual Studio Code copy it into the `<user home>/.vscode/extensions` folder and restart Code.
+* To share your extension with the world, read on https://code.visualstudio.com/docs about publishing an extension.
```

Comment:
```suggestion
## Install the extension

* To use the extension with Visual Studio Code or VSCodium, copy it into the `<user home>/.vscode/extensions` folder and restart the application.

To publish the extension so other can install and use it, refer to the [official documentation](https://code.visualstudio.com/docs)
```

Reply:
I assume a link would also work. Maybe it would be easier or more efficient to store the repo somewhere and link to it in your user home?

That way the user can update the repo without forgetting to copy it? Or should better kept separately?